pkg/build/release_push_build.go

```diff
@@ -18,36 +19,18 @@ var _ Stager = &ReleasePushBuild{}
 func (rpb *ReleasePushBuild) Stage(version string) error {
```

Comment:
We want the version to be used somewhere.
Ideally this is exactly what the version should be named in the final bucket.

I only see a VersionSuffix capability. https://godoc.org/k8s.io/release/pkg/release#PushBuildOptions
Is there a way to override the entire name?

Reply:
The main use case for that is running tests against uncommited local changes. 
For now we can probably make suffix work, I assume the auto-determined version comes from `hack/print-workspace-status.sh`? we do something similar (in `sourceVersion()` ) and append a kubetest2 uuid suffix.
We can do something like

```
VersionSuffix: strings.TrimPrefix(version, sourceVersion(repoRoot))
```

Reply:
Ah, no I added an option to overwrite the version, I think this makes sense.

Ref: https://github.com/kubernetes/release/pull/1644